packages/base-map/src/index.tsx

```diff
@@ -23,7 +23,9 @@ import MarkerWithPopup from "./MarkerWithPopup";
  */
 type Props = React.ComponentPropsWithoutRef<React.ElementType> & {
   /** A URL pointing to the vector tile specification which should be used as the main map.  */
-  baseLayer?: string;
+  baseLayer?: string | string[];
+  /** A list of names to match onto the base layers */
```

Comment:
Also say in the JSDoc that `baseLayerNames` is used only if multiple entries are defined for `baseLayer`.